gradle/libs.versions.toml

```diff
@@ -8,6 +8,5 @@ detekt-rules = { group = "io.gitlab.arturbosch.detekt", name = "detekt-rules", v
 detekt-formatting = { group = "io.gitlab.arturbosch.detekt", name = "detekt-formatting", version.ref = "detekt" }
 detekt-tooling = { group = "io.gitlab.arturbosch.detekt", name = "detekt-tooling", version.ref = "detekt" }
 detekt-testUtils = { group = "io.gitlab.arturbosch.detekt", name = "detekt-test-utils", version.ref = "detekt" }
-assertj-core = { group = "org.assertj", name = "assertj-core", version = "3.22.0" }
+assertj-core = { group = "org.assertj", name = "assertj-core", version = "3.23.1" }
 junit-jupiter = { group = "org.junit.jupiter", name = "junit-jupiter", version = "5.8.2" }
-slf4j-nop = { module = "org.slf4j:slf4j-nop", version = "1.7.36"  }
```

Comment:
Shall we really remove **slf4j.nop**? I thought this should prevent those log messages.

Reply:
As far as I know we now ship this dependency in detekt-core. It was necessary for detekt-formatting not to crash.
It does not crash anymore with 1.21.0-RC1.

Reply:
Yes, true. Now, I remember. This dependency is shipped with detekt itself.
=> Approved